starlark/support.go

```diff
@@ -15,6 +15,10 @@ import (
 	"go.starlark.net/starlarkstruct"
 )
 
+const (
+	ThreadUseSSHAgent = "crashd_use_agent"
```

Comment:
Why not use the identifier struct ? That way all identifier are located in one place (easier to refactor later).